frappe/installer.py

```diff
@@ -807,9 +807,9 @@ def get_old_backup_version(sql_file_path: str) -> Version | None:
 	"""
 	header = get_db_dump_header(sql_file_path).split("\n")
 	if match := re.search(r"Frappe (\d+\.\d+\.\d+)", header[0]):
-		backup_version = match[1]
+		return Version(match[1])
 
-	return Version(backup_version) if backup_version else None
+	return Version(backup_version) if match else None
```

Comment:
```suggestion
	return None
```
Github didn't let me edit both together on that comment, this can be simplified along with the above